packages/flex-plugin-utils-http/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "flex-plugin-utils-http",
+  "version": "4.5.0-beta.0",
+  "description": "Flex Plugins HTTP",
+  "keywords": [
+    "flex",
+    "flex plugin",
+    "flex plugin http",
+    "plugins"
+  ],
+  "homepage": "https://github.com/twilio/flex-plugin-builder",
+  "bugs": {
+    "url": "https://github.com/twilio/flex-plugin-builder/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/twilio/flex-plugin-builder.git"
+  },
+  "license": "MIT",
+  "author": "Kousha Talebian <k@ousha.me>",
```

Comment:
Can we use twilio email address here?

Reply:
Ya, let's do it in a followup email - my email is used in all of the package.json. I'll fix them all next.

Reply:
Also what email should we use?

Reply:
Lets switch to your twilio email address and please add flex-runtime@twilio.com too. Its better we change it now.